README.md

```diff
@@ -49,6 +49,7 @@ You will need `pyglet` to render some environments.
 ## What's new
 
 - 2019-02-09: Initial commit. Hello world :)
+- 2019-02-15: Added <a href="https://github.com/R-Sweke/gym-surfacecode">surfacecode</a> environment.
```

Comment:
I think it makes more sense to be from bottom up, so the newest changes are on top.

Reply:
I agree. Just commited a change addressing this.